include/fizzy/fizzy.h

```diff
@@ -275,6 +278,24 @@ uint8_t* fizzy_get_instance_memory_data(FizzyInstance* instance);
 /// @note    Function returns memory size regardless of whether memory is exported or not.
 size_t fizzy_get_instance_memory_size(FizzyInstance* instance);
 
+/// Find exported function by name.
+///
+/// @param  instance        Pointer to instance.
+/// @param  name            The function name. NULL-terminated string. Cannot be NULL.
+/// @param  out_function    Pointer to output struct to store the found function. Cannot be NULL.
+/// @param  out_context_ptr Pointer to opaque output pointer to a context associated with this
```

Comment:
Should also mention that in the case of failure (`returns false`), what will be the value of this (whether undefined or set to null) and whether that needs to be freed, or not.

Reply:
Added a sentence.